features/steps/partcad-cli/commands/version.py

```
@@ -0,0 +1,50 @@
+from behave import given, when, then
```

Ruff (0.8.0)

`behave.given` imported but unused

Remove unused import

(F401)